onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -315,6 +402,7 @@ onnxruntime::Status InvokeOp(_In_ const OrtKernelContext* context,
   auto ctx = reinterpret_cast<const OpKernelContext*>(context);
   AllocatorPtr allocator{};
   ORT_RETURN_IF_ERROR(ctx->GetTempSpaceAllocator(&allocator));
+  UpdateNode(reinterpret_cast<const onnxruntime::OpKernel*>(ort_op), input_count, output_count);
```

Comment:
It is because between the different calls to a kernel, the inputs and outputs account might change.
Take Conv for example, the same kernel might be called twice with bias or the first but not for the second.
If we do not change the number of inputs before calling, there is a chance of error with inside Conv::compute some code are trying to read the Node information for get an input number. 
The traditional way for calling into kernel does not have the trouble because the node behind is 1:1 mapping - meaning if two calls into the  call has different number of inputs, the node behind them must be different nodes in the graph.

Reply:
Conv is an exception. Normally, we should have stateless kernels

Reply:
They are stateless, but allowing for varied inputs. And when that happens, the node behind the kernel is preferred to be consistent.

Reply:
Each kernel should map to one node with defined set of inputs and outputs. What happens if you've 2 threads running the same kernel? One thread might have changed the counts without the other thread knowing about it.

Reply:
Switch to the approach that we are adding input/output count as arguments when creating the operator.
During invoke any change of input/output count will be treated as an error.